### PR TITLE
set has_numeric_value.range to decimal, remove QuantityValue.slot_usage.has_numeric_value.range

### PR DESCRIPTION
**Pull request by turbomam**
Waiting for review for @dwinston @PeopleMakeCulture and or @eecavanna

### CODE REVIEW

File: src/schema/attribute_values.yaml
Comment (dwinston) [inline src/schema/attribute_values.yaml:1]: Can you please remind me why the range cannot be string-valued (i.e. equivalent to a `xsd:decimal`-typed literal) in order to faithfully record numerical precision?

Comment (turbomam) [reply src/schema/attribute_values.yaml:1]: I hadn't thought about that!

Comment (turbomam) [reply src/schema/attribute_values.yaml:1]: https://linkml.io/linkml-model/latest/docs/Decimal/

**Comment by turbomam** [comment]
will change global range of `has_numeric_value` to `decimal` and remove QV's `slot_usage`

**Pull request merged**